RobotCode/snobot2018/src/org/snobot/elevator/IElevator.java

```diff
@@ -0,0 +1,26 @@
+package org.snobot.elevator;
+
+/**
+ * This is the interface for the elevator that allows someone to move the
+ * elevator up and down.
+ * 
+ * @author Josh
+ *
+ */
+public interface IElevator
+{
+    /**
+     * Gets the elevator height in inches
+     * 
+     * @return the elevator height in inches
+     */
+    double getHeight();
+
```

Comment:
Shouldn't the elevator interface also at least have a goUp() and goDown()? I'm not sure if I just can't see it or not

Reply:
We are using the joystick or the autonomous mode input.
-Josh Kline